### ISSUE TITLE
No longer usable since UI split out of parity?

### ISSUE THREAD

**Issue by jscinoz** [ethcore/parity-extension#123]
Is this extension deprecated? There does not appear to be any way to use it with Parity since the UI has been split out, as the extension still attempts to load the UI to extract an authentication token.

If this is the case, is there truly no longer any way to use Parity with dapps beyond those shipped with the new shell (e.g. MakerDao, Etherdelta, etc)?

Reproducing it is pretty trivial - run parity & parity-shell master, install the latest published parity extension = never connects to parity

**Comment by 5chdn** [comment]
The token is still available, @Tbaut @tomusdrw - or did we remove this, too, with the UI?

**Comment by Tbaut** [comment]
I don't think we removed it. However the extension can't connect to the node, even with `--force-ui --ws-origin all --jsonrpc-cors all`. Got a `The message port closed before a response was received.` in the extension's JS console.

This was with one of the latest master.

**Comment by fifinger** [comment]
I have just installed Parity, Parity UI and Chrome Extension on a new machine. Parity UI is running and sync is completed but the Chrome Extension can not connect. Strangely when I go into Extension options I see a green light (or yellow when syncing) next to Parity local node http://127.0.0.1:8180. When I click on the Parity button in the bookmark bar I only get "You are not connected to a local Parity Node". I tried to read up on the wiki but there is no info there.

**Comment by jscinoz** [comment]
Parity 1.10.8 with the `--force-ui` option is the last version that works with the extension at present.

**Comment by 5chdn** [comment]
1.10.9 and 1.11.6 do not work with `--force-ui` for you?

**Comment by Tbaut** [comment]
Same as what I wrote [before](https://github.com/paritytech/parity-extension/issues/123#issuecomment-404071641) on v2.0 with `parity --chain kovan --force-ui`

![image](https://user-images.githubusercontent.com/33178835/42630418-f479ec86-85d6-11e8-9910-a3f651a3ceda.png)

**Comment by tomusdrw** [comment]
@Tbaut could you check the debug console of background thread of the extension? go to `chrome:///extensions` and press `inspect` on the `background.html` in the extension

**Comment by Tbaut** [comment]
Sure:
Console:
![image](https://user-images.githubusercontent.com/33178835/42633695-8e0f10ec-85e1-11e8-88ed-2d9c05eeecc1.png)

Network:
![image](https://user-images.githubusercontent.com/33178835/42633756-b55be1de-85e1-11e8-8abf-45ca209857da.png)

The server on the port 8180 has been removed right?

**Comment by axelchalon** [comment]
(fwiw [this](https://github.com/parity-js/shell/pull/159) is how we check if the node is running in Parity UI)

**Comment by ilhanu** [comment]
I have the same with Parity 2.0.0 beta, the chrome extension doesn't connect to the node. 

Parityu UI (0.3.2.) connects well, what can be a quick fix for this problem ?

**Comment by 3X3Q** [comment]
Parity/browser integration is rather important for me to use dapps requiring full local nodes; seems like it might be for other people as well. Any way to elevate this issue?

**Comment by Tbaut** [comment]
It has been decided that the chrome extension will not be maintained anymore as Parity Ethereum is focusing on being a fast and reliable node, without any accessory. 

We advise to use Metamask connected to a local Parity Ethereum node to browse web3 Websites. Metamask supports importing accounts with a JSON keystore file that you can either export using Parity UI or find on your local machine at the following path:
  
-  OSX : `~/Library/Application\ Support/io.parity.ethereum/keys/`
-  Linux : `~/.local/share/io.parity.ethereum/keys/`
-  Windows: `%APPDATA%\Parity\Ethereum\keys`
